facts.html

```diff
@@ -284,5 +423,15 @@ <h3>SVG Template:</h3>
              transform="scale(381)"
              id="g1223" /></g></g></g></g></svg>
 </div>
+
+<div id='placeholder'></div>
```

Comment:
Here unfortunately i think we're conflicting with the new code from https://github.com/oshwa/certification-mark-generator/pull/6 - are you able to see if the code from that PR works with yours, or otherwise resolve the conflict? Thanks!

Reply:
Ah, i see the comment above now. fixing!